### ISSUE TITLE
Key to the Store has empty value

### ISSUE THREAD

**Issue by AmProsius** [AmProsius/gothic-1-community-patch#178]
**Describe the bug**
The Key to the Store has the label "Value" in the inventory, but the item has no value.

**Expected behavior**
The Key to the Store no longer has a "Value" label.

**Additional context**
The key has a value of 0 and therefor the value isn't displayed.  The German description of the key is *Lagerschlüssel*.

**Comment by AmProsius** [comment]
https://github.com/AmProsius/gothic-1-community-patch/blob/2b5e59952fbc8dacbade0a08317d9c519e5c6261/scriptbase/_work/Data/Scripts/Content/Items/MissionItems_1.d#L1172-L1183

changed to

```d
    TEXT[0]             = "Opens the main door of the store";
    ////COUNT[0]            = ;
    TEXT[1]             = "in the Ore Barons' cellar.";
    ////COUNT[1]            = ;
    //TEXT[2]               = "";
    //COUNT[2]          = ;
    //TEXT[3]           = "";
    //COUNT[3]          = ;
    //TEXT[4]               = "";
    ////COUNT[4]            = ;
    //TEXT[5]             = NAME_Value;
    //COUNT[5]            = value;
```

**Comment by szapp** [comment]
This bug will be resolved by #176.
